### PR TITLE
Fixes issue #7: Add text editors and create a test page with new editors.

### PR DESCRIPTION
**Pull request by klymenkoo**
This PR contains a fix for issue #7.

It adds a new dependency [React-Quill](https://github.com/zenoamaro/react-quill), which is a Text editor React component.

I've also created a test page under `CreativeCollab.jsx` component, where I placed two text editors and one kind of result container.

Here is how it looks:
cc

### CODE REVIEW

File: src/CreativeCollab.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+.story-container{
+    position: absolute;
+    border: 1px solid black;
+    padding-left: 1%;
+    bottom: 5%;
+    left: 20%;
+    width: 800px;
+    height:380px;
+}
```
Comment (SeanPrashad) [inline src/CreativeCollab.css]: nit: new line at end of file pretty please!

File: src/CreativeCollab.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                    <TextEditor handleChange={this.addTextFromSecondEditor}
+                        ref={instance => { this.secondEditor = instance; }}/>
+                </div>
+                <div className="story-container"  dangerouslySetInnerHTML={{ __html: this.state.story }} /> 
+            </React.Fragment>
+        );
+    }
+}
+
+export default CreativeCollab;
```
Comment (SeanPrashad) [inline src/CreativeCollab.jsx]: New line at EOF

File: src/TextEditor.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        return (
+            <ReactQuill value={this.state.text}
+                onChange={this.handleChange}
+                onBlur={this.props.handleChange}
+                style={{width: "500px", height: "300px"}} />
+        )
+    }
+}
+
+export default TextEditor;
```
Comment (SeanPrashad) [inline src/TextEditor.jsx]: New line at EOF

File: src/CreativeCollab.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        super();
+        this.state = {
+            story: ''
+        };
+
+        this.addTextFromFirstEditor = this.addTextFromFirstEditor.bind(this);
+        this.addTextFromSecondEditor = this.addTextFromSecondEditor.bind(this);
+    }
+
+    addTextFromFirstEditor(){
```
Comment (SeanPrashad) [inline src/CreativeCollab.jsx]: There's some duplicate logic here with `addTextFromFirstEditor ` and `addTextFromSecondEditor`. Let's refactor that in a future PR 🙌🏽

File: src/TextEditor.jsx (hunk 3 above)
Comment (klymenkoo) [reply src/TextEditor.jsx]: ✅

File: src/CreativeCollab.jsx (hunk 2 above)
Comment (klymenkoo) [reply src/CreativeCollab.jsx]: ✅

File: src/CreativeCollab.css (hunk 1 above)
Comment (klymenkoo) [reply src/CreativeCollab.css]: ✅

File: src/CreativeCollab.jsx (hunk 4 above)
Comment (klymenkoo) [reply src/CreativeCollab.jsx]: Agree. 
I've just refactored it, check the new commit

Comment (SeanPrashad) [reply src/CreativeCollab.jsx]: 👏

**Pull request merged**